pkg/api/resource/quantity_test.go

```diff
@@ -275,6 +282,7 @@ func TestQuantityString(t *testing.T) {
 		{Quantity{dec(3, 3), DecimalExponent}, "3e3"},
 		{Quantity{dec(3, 3), DecimalSI}, "3k"},
 		{Quantity{dec(0, 0), DecimalExponent}, "0"},
+		{Quantity{Amount: nil, Format: DecimalSI}, "0"},
```

Comment:
Actually, this was an outlier as I tried to find the cause.  I can remove the test if you want but this is the result of what we get from our conversion functions so I think the test is still good